### PR TITLE
Add fixed position pick app

### PR DESCRIPTION
**Pull request by RoboticsYY**
This PR intends to add fixed position pick app.

### CODE REVIEW

File: grasp_apps/fixed_position_pick/launch/fixed_position_pick.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+ur_test:
```
Comment (sharronliu) [inline grasp_apps/fixed_position_pick/launch/fixed_position_pick.yaml]: can you check why depends on ur_test?

File: grasp_apps/fixed_position_pick/package.xml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  <name>fixed_position_pick</name>
+  <version>0.5.0</version>
+  <description>A demo app for draw_x</description>
+  <author email="yu.yan@intel.com">Yu Yan</author>
+  <maintainer email="yu.yan@intel.com">Yu Yan</maintainer>
+  <license>Apache License 2.0</license>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+  <build_depend>rclcpp</build_depend>
+  <build_depend>Eigen3</build_depend>
```
Comment (sharronliu) [inline grasp_apps/fixed_position_pick/package.xml]: Can you check if Eigen3 is necessary?

File: grasp_apps/fixed_position_pick/launch/fixed_position_pick.yaml (hunk 1 above)
Comment (RoboticsYY) [reply grasp_apps/fixed_position_pick/launch/fixed_position_pick.yaml]: Similar to fixed_position_pick. I missed to change the name.

File: grasp_apps/fixed_position_pick/package.xml (hunk 2 above)
Comment (RoboticsYY) [reply grasp_apps/fixed_position_pick/package.xml]: Eigen3 is not necessary here.

**Pull request merged**